package.json

```diff
@@ -68,6 +68,7 @@
     "lint": "standard",
     "test": "npm run lint && npm run dependency-check && cross-env DEBUG=airtap* tape test",
     "test-local-phantom": "cross-env DEBUG=airtap* tape test/integration/tape-phantom.js",
+    "test-local-electron": "cross-env DEBUG=airtap* tape test/integration/tape-electron.js",
```

Comment:
Maybe we should have a `test-local` target which runs everything but the sauce tests.

Reply:
Good idea :)